lib/spack/spack/test/llnl/util/cpu.py

```diff
@@ -156,6 +157,7 @@ def test_architecture_family(target_name, expected_family):
     ('skylake', 'sse3'),
     ('power8', 'altivec'),
     ('broadwell', 'sse4.1'),
+    ('skylake', 'clflushopt'),
```

Comment:
This is fine to have here, but I want to clarify that it wouldn't have caught the issue you fixed for Mac OS detection. The `info` dictionary above is built for detection and the features in there are best-matched to the target we have in the JSON file. This unit test instead *starts* from one of the targets in the JSON file and checks that it has an expected feature.

Reply:
Fair enough, thanks.